.github/dependabot.yml

```diff
@@ -5,3 +5,18 @@ updates:
     directory: "/server"
     schedule:
       interval: "daily"
+  # yarn
+  - package-ecosystem: "npm"
+    directory: "/src"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-type: "all"
+    assignees:
+      - "davinov"
+      - "alice-sevin"
+      - "bloodstorms"
```

Comment:
This will be a pain to maintain if the squad changes, maybe we just keep an eye on it without mentioning anyone in particular?

Reply:
Also maybe reviewers is better than assignees?

Reply:
alright, let's remove that !